app/controllers/simple_smart_answers_controller.rb

```diff
@@ -40,6 +47,17 @@ def change_completed_question_path(question_number)
     smart_answer_path_for_responses(@flow_state.completed_questions[0...question_number], previous_response: @flow_state.completed_questions[question_number].slug)
   end
 
+  def artefact
```

Comment:
What's the difference between this and `fetch_artefact`?  We use both here, but they seem to do similar things